### DISCUSSION TITLE
Tooltip should have tooltip-none for responsive designs

### DISCUSSION THREAD

**Discussion by saulhoward** [saadeghi/daisyui#2253] [Ideas / feature requests]
I can make tooltips appear responsively at larger sizes:

```
lg:tooltip
```

But I can't have tooltips at only smaller sizes:

```
tooltip lg:tooltip-none
```

**Comment by saadeghi** [comment]
Tailwind CSS has responsive prefixes like `max-sm:`, `max-md:`, etc.  
```
max-lg:tooltip
```
